src/configSchema.ts

```diff
@@ -171,6 +173,20 @@ function transformBlocklist(blockList: string[], ctx: RefinementCtx) {
 					addZodIssue(blockRaw, ZodErrorMessages.blocklistRegex, ctx);
 				}
 				break;
+			case BlocklistType.CATEGORY:
+				if (blocklistValue.length === 0) {
+					logger.info(
+						`Blocklisting all torrents without a category due to empty ${blockRaw}`,
+					);
+				}
+				break;
+			case BlocklistType.TAG:
+				if (blocklistValue.length === 0) {
+					logger.info(
+						`Blocklisting all torrents without a tag due to empty ${blockRaw}`,
+					);
```

Comment:
kinda feel like empty array should not block anything, both intuitively and also because you can't combine it with other stuff - like if you want to block no category and also block books, this doesn't let you do that

 what about something like `[null]` or `[""]`?

Reply:
This is not when `blockList: []`, but for `blockList: ["category:", "tag:", "category:books"]`. The first two blocks torrents without categories or tags.

If a user tried the first two before this PR, config validation would have failed (or said they are using legacy linking with raw blocklist of `category:` or `tag:`)

Reply:
ohhhh I see yeah that makes sense then